.tool-versions.sample

```diff
@@ -1,2 +1,3 @@
-ruby 2.6.6
-nodejs 14.15.4
+ruby 2.6.8
+nodejs 14.20.0
+python 3.10.6 2.7.18
```

Comment:
we require python?

Reply:
ok, it is a mac thing

Reply:
I'm just concerned that new users may be confused and install python from asdf on fedora, which could be confusing later. But I don't know how to explain this well.

Reply:
Python: some packaged required it as a dependency, not sure if postgres or mysql so yes we require it. We could add a comment to the right saying "For M1 macs missing python".

Node 14 vs 12: good catch!! I need to use node 14 because of the new Mac chip but it should be 12.

Reply:
should we maybe upgrade to 14 for all. 12 not supported in Fedora 36 anymore. Also in RHEL8 I'm not sure or RHEL9.

Reply:
It's already planned: https://issues.redhat.com/browse/THREESCALE-7331
I'd stick with node 12 in our local env unless strictly necessary.